hibernate/cache-v51/src/test/java/org/infinispan/test/hibernate/cache/v51/util/BatchModeTransactionCoordinator.java

```diff
@@ -64,19 +64,15 @@ public void rollback() {
 
 		@Override
 		public TransactionStatus getStatus() {
-			try {
-            EmbeddedTransaction transaction = tm.getTransaction();
-				return transaction == null ? TransactionStatus.NOT_ACTIVE : StatusTranslator.translate(transaction.getStatus());
-			} catch (SystemException e) {
-				throw new RuntimeException(e);
-			}
+			EmbeddedTransaction transaction = tm.getTransaction();
+			return transaction == null ? TransactionStatus.NOT_ACTIVE : StatusTranslator.translate(transaction.getStatus());
```

Comment:
could this code be handled inside the StatusTranslator to avoid duplication ? just a suggestion

Reply:
I don't see how. What's your idea?
The `StatusTranslator` is hibernate code and it receives an `int`. I need to check if the transaction exists.